griffon/commands/entities/osidb.py

```diff
@@ -201,6 +214,324 @@ def create_flaw(ctx, **params):
+@click.option(
+    "--flaw-id",
+    "flaw_id",
+    help="Flaw CVE-ID or UUID.",
+    required=True,
+)
+@click.option("--uuid", "reference_uuid", help="Reference UUID.", required=True)
+@query_params_options(
+    entity="Flaw Reference",
+    endpoint_module=osidb_api_v1_flaws_references_retrieve,
+    options_overrides={
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(FlawReference))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(FlawReference))},
+        "include_meta_attr": {
+            "type": click.Choice(OSIDBService.get_meta_attr_fields(FlawReference))
+        },
+    },
+)
+@click.pass_context
+@progress_bar
+def get_flaw_reference(ctx, flaw_id, reference_uuid, **params):
+    params = multivalue_params_to_csv(params)
+
+    session = OSIDBService.create_session()
+    data = session.flaws.references.retrieve(flaw_id, reference_uuid, **params)
+    return cprint(data, ctx=ctx)
+
+
+@flaw_references.command(name="list")
+@click.option(
+    "--flaw-id",
+    "flaw_id",
+    help="Flaw CVE-ID or UUID.",
+    required=True,
+)
+@query_params_options(
+    entity="Flaw References",
+    endpoint_module=osidb_api_v1_flaws_references_list,
+    options_overrides={
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(FlawReference))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(FlawReference))},
+        "include_meta_attr": {
+            "type": click.Choice(OSIDBService.get_meta_attr_fields(FlawReference))
+        },
+    },
+)
+@click.pass_context
+@progress_bar
+def list_flaw_references(ctx, flaw_id, **params):
+    # TODO: handle pagination
+    # TODO: handle output
+    session = OSIDBService.create_session()
+
+    params = multivalue_params_to_csv(params)
+    data = session.flaws.references.retrieve_list(flaw_id, **params).results
+    return cprint(data, ctx=ctx)
+
+
+@flaw_references.command(name="create")
+@click.option(
+    "--flaw-id",
+    "flaw_id",
+    help="Flaw CVE-ID or UUID.",
+    required=True,
+)
+@request_body_options(
+    endpoint_module=osidb_api_v1_flaws_references_create,
+    exclude=["uuid", "created_dt"],
+)
+@click.pass_context
+def create_flaw_reference(ctx, flaw_id, **params):
+    request_body_type = getattr(osidb_api_v1_flaws_references_create, "REQUEST_BODY_TYPE", None)
+    if request_body_type is None:
+        raise click.ClickException(
+            "No request body template for Flaw Reference create. "
+            "Is correct version of osidb-bindings installed?"
+        )
+
+    fields = filter_request_fields(
+        request_body_type.get_fields(),
+        exclude=["uuid", "created_dt"],
+    )
+    params = multivalue_params_to_csv(params)
+
+    session = OSIDBService.create_session()
+
+    data = {field: "" for field in fields}
+    data.update((field, value) for field, value in params.items() if value is not None)
+
+    if ctx.obj["EDITOR"]:
```

Comment:
does no-editor flag interact with EDITOR ?

Reply:
Yes, that is basically just a boolean flag to tell Griffon whether editor prompt should happen. Actual editor (vi, emacs, etc.) is then obtained from the config.